README.adoc

```diff
@@ -182,6 +183,32 @@ sandbox = true
 extra-sandbox-paths = /System/Library/Frameworks /System/Library/PrivateFrameworks /usr/lib /private/tmp /private/var/tmp /usr/bin/env
 ----
 
+[[nix-macos-trusted-non-root-user]]
+=== Nix macOS Trusted Non Root
+Nix daemon runs under `root` user as it is explained in https://nixos.org/nix/manual/#ssec-multi-user[trusted user]. This would not allow us to run `nix-shell` as a non `root` user, using binary caches
```

Comment:
I don't see that section anymore, I suppose it was deleted and the doc was updated (maybe nix itself as well).

Personally I have a different setup without adding configurations with trusted-users. Maybe it was a special requirement for some special macOS version.

Reply:
Thanks @ak3n for your comment. It would be related to macOS version. Now i have migrated to a Lenovo with Linux OS, so I cannot test again this anymore. When I had configured this, I was in a macOS Mojave version and I did the installation of Plutus from scratch. Any previous Plutus version or anything. 

I would be good to do some similar setup from scratch to see if this PR is still needed but i cannot reproduce it now since i dont have anymore mac :disappointed: